lib/phlex/sgml.rb

```diff
@@ -5,6 +5,17 @@ class Phlex::SGML
 	UNSAFE_ATTRIBUTES = Set.new(%w[srcdoc sandbox http-equiv]).freeze
 	REF_ATTRIBUTES = Set.new(%w[href src action formaction lowsrc dynsrc background ping]).freeze
 
+	ERBCompiler = ERB::Compiler.new("<>").tap do |compiler|
+		compiler.pre_cmd    = [""]
+		compiler.put_cmd    = "@_state.buffer.<<"
+		compiler.insert_cmd = "__implicit_output__"
+		compiler.post_cmd   = ["nil"]
+
+		def compiler.add_insert_cmd(out, content)
+			out.push("#{@insert_cmd}((#{content}))")
```

Comment:
We define this because the original implementation called `to_s` on everything which meant we couldn’t have different behaviour for different things.